.travis.yml

```diff
@@ -1,7 +1,4 @@
 language: node_js
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
-  - "iojs"
-  - "iojs-v1.0.4"
+  - 8
```

Comment:
7 is already at EOL status - https://github.com/nodejs/LTS
is there a point in adding 7?